median/report_codec.go

```diff
@@ -0,0 +1,44 @@
+package median
+
+import (
+	"context"
+	"errors"
+	"math/big"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+const TypeName = "MedianReport"
```

Comment:
Oh could this be unexported too?
```suggestion
const typeName = "MedianReport"
```